BUILD.gn

```diff
@@ -915,6 +915,26 @@ dist_zip("electron_ffmpeg_zip") {
   ]
 }
 
+dist_zip("electron_mksnapshot_zip") {
+  data_deps = [
+    ":electron_mksnapshot",
```

Comment:
You need to define the target `electron_mksnapshot`, which should just be a copy task like out licenses.